app/models/foreman_docker/docker.rb

```diff
@@ -131,17 +132,19 @@ def bootstrap(args)
     end
 
     def client
-      @client ||= ::Fog::Compute.new(
+      cli = ::Fog::Compute.new(
```

Comment:
Actually it looks like it supports multiple servers. Working on a fix for fog:

https://github.com/daviddavis/fog/commit/b428f2b4b4f6dd010252f59ed688f300ed4e1466